oshc/oshc/settings.py

```diff
@@ -105,3 +120,12 @@
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'main/'),
 )
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_USE_TLS = True
```

Comment:
Change this and following lines to `os.getEnv('EMAIL_USE_TLS)`. This way it is easier to change the email server and related settings.